spec/dummy/public/doc/openapi-3-v1.0.0.json

```diff
@@ -263,7 +263,7 @@
         }
       }
     },
-    "/source_types/{id}/sources": {
```

Comment:
This no longer matches the documented parameter name

Reply:
👍 

- Updated the openapi json to:
   -  properly define the source_type_id
   -  remove entries not defined in the schema (or used in specs)
   -  fixed missing item type definition for the array field.
This openapi spec now passes the swagger editor validation.